src/components/chipsInput/index.js

```diff
@@ -96,27 +96,32 @@ class ChipsInput extends Component {
       value: props.value,
       tags: _.cloneDeep(props.tags) || [],
       tagIndexToRemove: undefined,
-      initialTags: props.tags
+      initialTags: props.tags,
+      inputRef: undefined
     };
   }
 
   componentDidMount() {
     if (Constants.isAndroid) {
-      const textInputHandle = ReactNative.findNodeHandle(this.input);
+      const textInputHandle = ReactNative.findNodeHandle(this.state.inputRef);
```

Comment:
The ref should not be on the state. See my comment in `getDerivedStateFromProps`.